### DISCUSSION TITLE
Arrange Repository Overview Folders to own liking?

### DISCUSSION THREAD

**Discussion by EliasDrozd** [community/community#80991] [New to GitHub]
Hi there mates,

I would like to have my private Repository to have a new arrangement. For example Code is at the bottom because it was created last, but I would like to have it always placed on top of the other folder I have created.

Is there a way to "move" these folders around and save their arrangement location ?

With Kind regards!

**Comment by queenofcorgis** [comment]
Hi there 😃 ! 

I dig some investigating and unfortunately there is not a method to customize this arrangement.
